app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def tip_rename
```

Comment:
Interesting. Let's approach this from the perspective of the `Tip` class so that we know where to find it in our code (currently it's hidden away). Perhaps something like `Tip.alias`. If none is set, it defaults to the name of the class. Otherwise it uses the alias.